tests/dataproviders/config.go

```diff
@@ -0,0 +1,190 @@
+package dataproviders
+
+import "fmt"
+
+type ClickhouseFieldRaw struct {
+	ValueType string                 `json:"valueType"`
+	Config    map[string]interface{} `json:"config"`
+}
+
+type ClickhouseInsertJobRaw struct {
+	BatchSize   int64                         `json:"batchSize"`
+	TableName   string                        `json:"tableName"`
+	Partitions  int                           `json:"partitions"`
+	PartitionId int                           `json:"partitionId"`
+	Config      map[string]ClickhouseFieldRaw `json:"config"`
+}
+
+func (raw ClickhouseInsertJobRaw) Into(result *ClickhouseInsertJob) error {
```

Comment:
Question: Why was receiver ClickhouseInsertJobRaw not passed by reference

Reply:
It did not. Changed.